docs/sources/administration/configuration.md

```diff
@@ -1505,3 +1505,8 @@ Set this to `true` to have date formats automatically derived from your browser
 ### default_timezone
 
 Used as the default time zone for user preferences. Can be either `browser` for the browser local time zone or a time zone name from the IANA Time Zone database, such as `UTC` or `Europe/Amsterdam`.
+
+## [expressions]
```

Comment:
note to self: can cross link https://github.com/grafana/grafana/pull/30524/files once that is merged (so I imagine it will be in that PR, not this one).